src/language/typescript/common/data/serialized-type.ts

```diff
@@ -180,7 +180,7 @@ export const getSerializedIntersectionType = (serialized: NonEmptyArray<Serializ
 	} else {
 		const intercalated = intercalateSerializedTypes(serializedType(' & ', ',', [], []), serialized);
 		return serializedType(
-			intercalated.type,
+			`(${intercalated.type})`,
```

Comment:
But do we need this for `&`? It has always higher priority than `|` and it is associative.

Reply:
I guess you're right, fixed